### PR TITLE
Assignment_Third

### PR DESCRIPTION
**Pull request by Ganeshbabu18**

### CODE REVIEW

File: Get_all_brand.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+def test_api_status():
+    response = requests.get(API_URL)
+    assert response.status_code == 200
+
+
+def test_api_content():
+    response = requests.get(API_URL)
+    assert response.headers['Content-Type'] == 'application/json'
```
Comment (amitsahu8) [inline Get_all_brand.py]: verify the content in the response

File: Get_all_brand.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import requests
+import pytest
+
+API_URL = 'https://automationexercise.com/api/brandsList'
```
Comment (amitsahu8) [inline Get_all_brand.py]: variable should be camelcase

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: Get_all_brand.py (hunk 1 above)
Comment (Ganeshbabu18) [reply Get_all_brand.py]: @amitsahu8 Changes made. Kindly review it.

File: Get_all_brand.py (hunk 2 above)
Comment (Ganeshbabu18) [reply Get_all_brand.py]: @amitsahu8 Changes made. Kindly review it.

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**